book/hardware-batterybay.md

```diff
@@ -10,51 +10,44 @@ For information on interfacing with the battery, please visit the [Battery Inter
 
 To interface with Solo there are a few crucial interface points to pay attention to
 
-> <sub>* Interlocking with the battery tray
-
-> <sub>* Interfacing the electrical connection
-
-> <sub>* Using the release button
-
-> <sub>* Keep out zones for accessory payloads and props
+* Interlocking with the battery tray
+* Interfacing the electrical connection
+* Using the release button
+* Keep out zones for accessory payloads and props
 
 Most all of these interfaces can be found in the battery bay envelope cad file (link to come).
 
 Any battery options operating in the battery bay should minor approximately the same specs of the current battery:
 
```

Comment:
spelling - should "mirror" not "minor"

What about other specs - discharge rates? Is the weight important?